server.js

```diff
@@ -47,6 +47,8 @@ app.set('view engine', 'handlebars');
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'views')));
```

Comment:
What effect does this have? I commented it out after pulling down the changes and can't notice a difference.